lib/InteractiveViewer.ml

```diff
@@ -36,6 +36,10 @@ let view (patches : Patch.t list) =
       Lwd.set curr_scroll_state { state with position = state.W.bound }
     else Lwd.set curr_scroll_state state
   in
+  let project_op (op : Patch.t Zipper.t -> Nottui.ui) : ui Lwd.t =
+    let$ z_patches = Lwd.get z_patches_var in
+    op z_patches
```

Comment:
I think you rewrote `Lwd.map`:
```
          Lwd.map (Lwd.get z_patches_var) ~f:OperationView.operation_info;
```

Reply:
To clarify: `let$` is an alias to `Lwd.map`. This can be written in a shorter way without introducing a local function.

`Lwd.get` "observes" a variable, it returns a `Lwd.t` that changes every time the variable is changed.